packages/rxtranscribe/demo/runDemo.js

```diff
@@ -1,21 +1,49 @@
-import prompt from 'prompt';
```

Comment:
⬇️ This file is a demo for the speech-to-text pipelines.  It can be run from the package directory as `yarn demo:run` and then it has a console wizard that accepts inputs from the user.  Alternatively, options can also be passed in like `yarn demo:run --region us-east-1 --modelDir $HOME/Downloads/deepspeech-0.6.1-models`